### DISCUSSION TITLE
Docs: Deduplicate information in README.md vs. in profile/README.md?

### DISCUSSION THREAD

**Discussion by pbodnar** [eclipse-platform/.github#29] [Ideas]
While I understand,  _for some technical reasons_, there might be 2 separate READMEs necessary:

1. https://github.com/eclipse-platform/.github/blob/main/profile/README.md - displayed at https://github.com/eclipse-platform/
2. https://github.com/eclipse-platform/.github/blob/main/README.md - displayed at https://github.com/eclipse-platform/.github/

... I don't think it is really a good idea to have them like this, because:

1. They duplicate information.
2. There are inconsistencies between them, not sure if all are intentional, or not.
3. This might confuse readers while browsing the GitHub contents and getting to one or the other README "randomly".
4. This is hard to maintain.

What do you think? Is it possible to have just 1 README, or how to approach this problem?

**Comment by mickaelistria** [comment]
If there is only 1 README, it means either there is a description on https://github.com/eclipse-platform/ xor on https://github.com/eclipse-platform/.github/ , it can't be shown on both, can it?

**Reply by pbodnar** [reply]
According to GitHub's page [Customizing your organization's profile](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile#adding-a-public-organization-profile-readme) it looks like there is no customization possible regarding the organization's profile README.md location. That pretty much limits solutions available...

**Comment by pbodnar** [comment]
It looks like the de-duplication of the READMEs was done: https://github.com/eclipse-platform/.github/commit/b95aa9ab982c4f58df19fe39c25785ea57bdd8ad. Thanks to @HeikoKlare. 👍
